docs/first-order/current_source_density.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>Briefly chatted with Josh. Couple more thoughts: </p><ul><li>We are likely looking at hippocampal oscillations here that are very strong. It would be important to add the relevant dimensions along the y axis so that we know how deep those signals are. </li><li>Apparently, seeing CSD sink in the cortex is hard, so you need to average tens of trials. So it is likely that we need to select a repeating visual stimulus, and do averages of those trials to help signal peaks up against the background. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>